input/pagecontent/general-requirements.md

```diff
@@ -315,28 +143,40 @@ The full set of sub-elements is visible in the "Key Elements Table" or "Snapshot
 
 
 ##### Must Support - Primitive Elements
-Primitive elements are single elements with a primitive value. If they are marked as *Must Support*, then the AU Core Responder **SHALL** be capable of providing the element value to meet the *Must Support* requirement. AU Core Requestors **SHALL** be capable of processing a resource with all elements to meet the *Must Support* requirement.
+Primitive elements are single elements with a primitive value. If the primitive element is labelled as *Must Support*, then the 
+- AU Core Responder **SHALL** correctly populate the element if a value is known. 
+- AU Core Requesters **SHALL** accept resource without error if element present containing any valid value.
```

Comment:
I find this hard to read - but that may be me only. Suggest chaning to something like:  
- AU Core Requesters **SHALL** accept _a_ resource without error if _the_ element is present _and contains_ ~~containing~~ any valid value.

Reply:
is replaced with the exact text from the FHIR standard "resources containing any valid value for the element without error" also fixed equivalent sentence under the section "Must Support - Resource References"